images/harbor-database/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM amazeeio/postgres:v1.1.4
```

Comment:
please use the same way we use in other images already:
https://github.com/amazeeio/lagoon/blob/master/images/postgres-drupal/Dockerfile#L1-L2

plus we also need to add a dependency of the images to the Makefile, example:
https://github.com/amazeeio/lagoon/blob/dedicated-test-containers/Makefile#L168